docs/source/_static/custom.css

```diff
@@ -1,3 +1,24 @@
 .section dl.py:not(:first-of-type) {
     margin-top: 1.5em;
 }
+
+.strike {
+    text-decoration: line-through;
+}
+
+.strong {
+    font-weight: bold;
+}
+
+.em {
+    font-style: italic;
+}
```

Comment:
Why name these classes the HTML tag name? Why not `.bold` and `.italic`?

Reply:
I'll happily change that, I don't have a strong opinion.